vibi-dpu/src/github/auth.rs

```diff
@@ -19,34 +18,68 @@ struct Claims {
     iss: String,
 }
 
-fn generate_jwt(github_app_id: &str) -> Result<String, Box<dyn std::error::Error>> {
-    let pem_file_path = "/tmp/repoprofiler_private.pem";
-    let pem_data = fs::read(pem_file_path)?;
-
+fn generate_jwt(github_app_id: &str) -> Option<String> {
+    let pem_file_path = "/app/repoprofiler_private.pem";
+    let pem_data = match fs::read(pem_file_path) {
+        Ok(data) => data,
+        Err(e) => {
+            println!("Error reading pem file: {}", e);
+            return None;
+        },
+    };
```

Comment:
Don't use match for anything except debugging. You should read the result of fs::read(pem_file) in a variable, check if that variable is_err(), print the error and return . if not, the rest of the code follows. This helps in keeping the code clean (separation of logic and error handling). For reference, you can see any function in gitops.rs, like - `get_excluded_files`

Reply:
done!